suites/pacific/cephadm/tier-1_ssl_dashboard_port.yaml

```diff
@@ -62,6 +62,8 @@ tests:
       desc: installation of ceph pre-requisites
       module: install_prereq.py
       abort-on-fail: true
+      config:
+        is_production: true
```

Comment:
@manasagowri why are we setting this config in this suite?

Reply:
That I had added during testing, since there was issue with stage subscription. Thanks Sunil for pointing it out. I have removed it now.